lib/floe/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Floe
-  VERSION = "0.9.0".freeze
+  VERSION = "0.9.0"
```

Comment:
This one surprised me - I thought rubocop enforce freezing all constants 😕

Reply:
Oh duh, there's a frozen string literal at the top (I never liked those, and almost think that's the actual fix we should do here).